x-pack/plugins/lens/public/editor_frame_service/editor_frame/config_panel/dimension_container.scss

```diff
@@ -16,6 +16,8 @@
 
   @include euiBreakpoint('xs', 's', 'm') {
     @include euiFlyout;
+    left: 10vw;
```

Comment:
This works, but it seems to be overriding some default EUI behavior that should have this flyout working in the desired manner. The issue seems to be the `left: 0` on line 9 above. My guess is that line 9 seeks to ensure the flyout covers the full side panel as opposed to setting a fixed width/large flyout size. 

@MichaelMarcialis may have some better ideas, but it seems moving the `left: 0` into the `'l', 'xl'` breakbpoint will allow the flyout to honor the default EUI stylings for small screens.

Reply:
@ryankeairns' analysis is correct. At the same time, I'm not super keen on showing the rest of the Lens interface in the background when the configuration flyouts are open on smaller viewport sizes (looks a bit overwhelming and messy). To address both, I've put together a quick [design PR](https://github.com/dej611/kibana/pull/14) for you @dej611. Have a look when you get a moment, and feel free to merge in.